lib/database_connection.rb

```diff
@@ -0,0 +1,11 @@
+require 'pg'
+
+class DatabaseConnection
+  def self.setup(dbname)
+    @connection = PG.connect(dbname: dbname)
+  end
+
+  def self.query(sql)
+    @connection.exec(sql)
+  end
+end
```

Comment:
Great user of SRP, having a DatabaseConnection class makes the files cleaner - 100%!